pywb/rewrite/regex_rewriters.py

```diff
@@ -16,6 +16,19 @@ def remove_https(string, _):
     def replace_str(replacer, match='this'):
         return lambda x, _: x.replace(match, replacer)
 
+    @staticmethod
+    def replace_prefix_from(prefix, match):
+        def do_replace(x, _):
+            start = x.find(match)
+            if start == 0:
+                return prefix
+            if start > 0:
+                return x[:start] + prefix
```

Comment:
This could be simplified to:

```python
if start >= 0:
    return x[:start] + prefix
```
since a `[:0]` slice of any string is just an empty string. Arguably it's a bit more readable as-is though, so maybe just leave it?

Reply:
yeah, was thinking about readability, also ported it from:
https://github.com/webrecorder/wabac.js/blob/main/src/rewrite/jsrewriter.js#L38
to have some parity. so perhaps keep as is for now, unless changed in both places?